src/logger.js

```diff
@@ -17,12 +17,12 @@ class Logger {
     this.logger.info(msg);
   }
 
-  debug(...args) {
+  debug(data, ...args) {
     const msg = args.length <= 1 ? args[0] : args;
     this.logger.debug(data, msg);
```

Comment:
This line was a part of the codebase before my PR, `npm t` fails with the following message 
**_"22:23  error  'data' is not defined  no-undef"_**
So we have to either just call `this.logger.debug(msg);`, or get the `data` as an argument

Reply:
Yes thanks! We need to notify @nechaido to avoid this bug in other implementations/versions of this codebase

Reply:
Consider him notified.